### ISSUE TITLE
AVT issues in TypeLayout

### ISSUE THREAD

**Issue by joemc11** [carbon-design-system/ibm-security#944]
[RuleTactics.test.txt](https://github.com/carbon-design-system/ibm-security/files/6366352/RuleTactics.test.txt)

## Bug - 
We are unable to upgrade to version 1.40.0 as TypeLayout has accessibility issues, causing our AVT tests to fail
Version 1.39.0 works well

### Description

![]()

<img width="878" alt="Screenshot 2021-04-23 at 16 19 12" src="https://user-images.githubusercontent.com/14826627/115893044-ab67a480-a44f-11eb-8f09-8fbb8b0954cf.png">

#### Expected behavior

No accessibility issues for TypeLayout component

#### Actual behavior

Rule violation: #aria-required-children [critical]

### Steps for reproducing
Run attached unit test which includes markup from storybook https://github.com/carbon-design-system/ibm-security/blob/dev/src/components/TypeLayout/TypeLayout.stories.js

1. 

### Affected browsers

[What's my browser?](http://www.whatsmyua.com) and [browserl.ist supported browsers](https://github.com/carbon-design-system/ibm-security/blob/master/package.json#L174-L178)

- [ ]  / 
- [ ]  / 
- [ ]  / 

### Optional information

**Version -**

**Comment by SimonFinney** [comment]
Blocked by an upgrade to Carbon `v10.33` containing https://github.com/carbon-design-system/carbon/pull/8172
